bee-api/bee-rest-api/Cargo.toml

```diff
@@ -16,6 +16,7 @@ bee-message = { version = "0.2.0", path = "../../bee-message", default-features
 bee-protocol = { version = "0.1.1", path = "../../bee-protocol", default-features = false, optional = true }
 
 hex = { version = "0.4.3", default-features = false }
+primitive-types = { version = "0.10.1", default-features = false, features = [ "serde" ] }
```

Comment:
Where is this used?

Reply:
TryFrom<&NativeTokenDto> for NativeToken is using it now